### PR TITLE
README updated with contribution merge policy proposal

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

**Comment by vorburger** [comment]
@ctron @maggu2810 OK for you? @kaikreuzer any feedback based on your experience?

**Comment by maggu2810** [comment]
Okay for me. Reading the EEA syntax is okay if you do it some time.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ To automatically Enable Annotation-based Null Analysis in the Eclipse Project Pr
 
 This project aims to develop an active community of contributors, and not remain controlled by a single person.  Anyone making 3 intelligent contributions to this repo may ask to be promoted from a contributor to a committer with full write access by opening an issue requesting it.  (We reserve the right to re-remove committers in exceptional circumstances, and after long periods of inactivity.)
 
+We intend to liberally and quickly merge any contributions with additions to EEA, and avoid delays due to lengthy reviews 
+(which are anyway [kind of difficult to do rapidly and at scale until we solve issue #16](https://github.com/lastnpe/eclipse-null-eea-augments/issues/16)), 
+based on the idea that having an EEA that can be tested is better than none.
+
+We intend to spend more time and request community feedback from engaged previous contributors on any proposed changes to existing EEA in this repo.  When making contributions with changes, please explain what is wrong in the current version in the commit message.
+
+We generally do not "self merge", but let other commiters merge our own changes.
```
Comment (kaikreuzer) [inline README.md]: typo: committers

Comment (vorburger) [reply README.md]: @kaikreuzer thanks for spotting - fixed! (I'll self merge this now, given the x3 approvals.)

**Pull request merged**